packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -586,35 +592,36 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
     }
   }
 
-  Future<List<RemoteDiagnosticsNode>> _childrenFuture;
-  List<RemoteDiagnosticsNode> _children;
+  Future<List<RemoteDiagnosticsNode>>? _childrenFuture;
+  List<RemoteDiagnosticsNode>? _children;
 
   /// Reference the actual Dart DiagnosticsNode object this object is referencing.
   InspectorInstanceRef get dartDiagnosticRef {
-    return InspectorInstanceRef(json['objectId']);
+    return InspectorInstanceRef(json['objectId'] as String?);
   }
 
   /// Properties to show inline in the widget tree.
-  List<RemoteDiagnosticsNode> get inlineProperties {
+  List<RemoteDiagnosticsNode>? get inlineProperties {
     if (cachedProperties == null) {
       cachedProperties = [];
       if (json.containsKey('properties')) {
-        final List<Object> jsonArray = json['properties'];
-        for (Map<String, Object> element in jsonArray) {
-          cachedProperties.add(
+        final List<Object> jsonArray = json['properties'] as List<Object>;
+        for (Map<String, Object> element
```

Comment:
remove LHS type and use var in for loop instead of explicit type

Reply:
done